source/features/global-search-filters.tsx

```diff
@@ -1,54 +0,0 @@
-import React from 'dom-chef';
-import {XIcon} from '@primer/octicons-react';
-import elementReady from 'element-ready';
-import * as pageDetect from 'github-url-detection';
-
-import features from '../feature-manager';
-import SearchQuery from '../github-helpers/search-query';
-import {getUsername} from '../github-helpers';
-
-async function init(): Promise<void> {
-	const filters = [
-		['Forks', 'fork:true'],
```

Comment:
Missing 🥲

Reply:
This is supported since a while ago:

https://github.com/orgs/community/discussions/13126#discussioncomment-5294539

Not in the sidebar though (should we add it?)